src/api/schemas/extendedQueryNode.graphql

```diff
@@ -63,28 +63,34 @@ enum VideoMediaEncoding {
   Theroa_Vorbis
 }
 
-# Apparently there are lots of different Creative Commons licenses,
-# read about all here https://creativecommons.org/licenses/,
-# I haven't had the time.
-enum CreativeCommonsVersion {
-  CC_BY
-  CC_BY_SA
-  CC_BY_ND
-  CC_BY_NC
-  CC_BY_NC_SA
-  CC_BY_NC_ND
-}
-
-type CreativeCommonsLicense {
-  version: CreativeCommonsVersion
+type KnownLicense {
+  code: String!
+  name: String
+  description: String
+  url: String
 }
 
 type UserDefinedLicense {
-  text: String!
+  content: String!
 }
 
-union License = UserDefinedLicense | CreativeCommonsLicense
+union License = UserDefinedLicense | KnownLicense
 
+type LicenseEntity {
+  id: ID!
```

Comment:
Not sure if we should put all the fields here that the query node exposes. In general so far I've tried only including fields that we actually use. I guess things like `createdById` can be removed once the query node implementation changes so we shouldn't rely on them.

Reply:
Good point, I will leave only the fields that we actually use